services/fdb/fdb.proto

```diff
@@ -360,6 +553,9 @@ message FDBCLITransaction {
 
 message FDBCLIUnknownAction {}
 
+// FDBCLIRequest defines the fdbcli command(s) to run.
+// This will replicate fdbcli --exec "command(s)" (where N commands are
+// separated by ;) NEXT: 19 DO NOT USE: 11, 12, 13
```

Comment:
you can mark 11,12,13 as 'reserved' to prevent re-use: https://developers.google.com/protocol-buffers/docs/proto3#reserved

Reply:
Done